sig-lifecycle/charter.md

```diff
@@ -0,0 +1,35 @@
+# SIG Lifecycle Charter
+
+## Scope
+
+The scope of SIG Lifecycle is currently limited to the implementation and graduation of the `instancetype.kubevirt.io` API and CRDs to `v1`.
+
+In the future it is hoped to increase the scope of the SIG to other aspects of a `VirtualMachine` lifecycle within KubeVirt as part of the process to break up the responsibilities of the current default `sig-compute`.
+
+### In scope
+
+- The `instancetype.kubevirt.io` API and CRDs
+- `pkg/instancetype`
+- `pkg/virt-operator/resource/generate/components/data/common-clusterinstancetypes-bundle.yaml`
+- `pkg/virt-operator/resource/generate/components/data/common-clusterpreferences-bundle.yaml`
+- `tests/libinstancetype`
+- `tests/instancetype`
+- `kubevirt/common-instancetypes`
```

Comment:
Add `pkg/virtctl/create/instancetype` and `pkg/virtctl/create/preference`?

Reply:
And what about:
- `pkg/virt-api/webhooks/validating-webhook/admitters/instancetype-admitter.go`
- `pkg/virt-operator/resource/apply/instancetypes.go`
- `pkg/virt-operator/resource/generate/components/instancetypes.go`

There are also some more places but I'm not sure we need to list them all.

Reply:
Yeah don't know if we need to be explicit here but I guess it doesn't hurt for now. Added.